knotx-core/knotx-service-adapter/src/main/java/com/cognifide/knotx/core/serviceadapter/http/HttpClientFacade.java

```diff
@@ -0,0 +1,58 @@
+/*
+ * Knot.x - Sample App with Mock service
+ *
+ * Copyright (C) 2016 Cognifide Limited
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.cognifide.knotx.core.serviceadapter.http;
```

Comment:
We don't need to have this class in separate package.
This introduces cyclic dependency between packages.
Maybe some of classes could be package-scoped?